Troubleshooting_Ipv6.md

```diff
@@ -0,0 +1,273 @@
+* 3 Master nodes. 
+
+Before starting the installation, we need to set up a DNSmasq conf file which will assign IPv6 IP's to all the nodes on `eno3` interface.A sample file is provided below.  
+
+* A Jump Host should have one interface say `eno1` on Ipv4(Public Network), `eno3` on **Private VLAN** this will be used to provide IPv6 IP's on the isolated VLAN
+
+```
+domain-needed
+bogus-priv
+strict-order
+#expand-hosts
+bind-dynamic
+dhcp-sequential-ip
+dhcp-authoritative
+dhcp-fqdn
+log-queries
+domain=<Your_Domain_Name>
+resolv-file=/etc/resolv.conf.upstream
+interface=eno3
+enable-ra
+server=<Dns_Server>
+
+dhcp-range=<Ipv6-Range-1>,<IPv6-Range-2>,<Network-Subnet> 
+dhcp-option=option6:dns-server,[Jump-Hosts-IPv6-IP]
+dhcp-option=option6:ntp-server,[::]   
+
+local=/<Your_FQDN>/
+
+#static addresses
+address=/.apps.<Cluster-Name>.<Domain-Name>/<Ipv6-IP>
+address=/api.<Cluster-Name>.<Domain-Name>/<Ipv6-IP>
+address=/ns1.<Cluster-Name>.<Domain-Name>/<Ipv6-IP>
+address=/registry.<Cluster-Name>.<Domain-Name>/**<IPv6-IP-Jump-Host>**
+
+#reserved dynamic addresses
+dhcp-host=<MAC-Address-eno3-Interface>,master-0.<Cluster-Name>.<Domain-Name>,[IPv6-IP-master-0]
+dhcp-host=<MAC-Address-eno3-Interface>,master-1.<Cluster-Name>.<Domain-Name>,[IPv6-IP-master-1]
+dhcp-host=<MAC-Address-eno3-Interface>,master-2.<Cluster-Name>.<Domain-Name>,[IPv6-IP-master-1]
+dhcp-host=<MAC-Address-eno3-Interface>,worker-0.<Cluster-Name>.<Domain-Name>,[IPv6-IP-worker-0]
+```
+
+A sample `etc/resolv.conf.upstream` file is as shown below
+
+```
+# Generated by NetworkManager
+search <Cluster-Name>.<Domain-Name>
+nameserver <IPv6-IP-Jump-Host>
+```
+
+A sample `/etc/hosts` file should be configured as shown
+
+```
+<IPv6-IP-Jump-Host> registry.<Cluster-Name>.<Domain-Name>
+```
+
+**Note:** After all the above file changes, make sure you can resolve all the above hosts using nslookup and dig commands.
+
+## Create a Disconnected Registry
+
+* [Create a Disconnected Registry](https://docs.openshift.com/container-platform/4.3/installing/install_config/installing-restricted-networks-preparations.html#installing-restricted-networks-preparations)
+
+**Note:** While Mirroring the registry, please make sure to add the additional registry.svc.io token to your pull-secret.txt file, else the mirroring process fails. [Add-Token](https://docs.openshift.com/container-platform/4.3/installing/install_config/installing-restricted-networks-preparations.html#Procedure) 
+
+After the registry is mirrored, confirm that you can access this in your disconnected environment
+
+```
+# curl -k -6 -u kni:kni https://registry.<Cluster-Name>.<Domain-Name>:5000/v2/_catalog
+{"repositories":["ocp4"]}
+```
+
+Now, we need to verify the images are served from local http service(either create an nginx local server or httpd container service) 
+
+```
+# curl -I -s http://<Jump-Host>.<Domain-Name>
+HTTP/1.1 200 OK
+
+# curl -I -s http://<Jump-Host>.<Domain-Name>/$RHCOS_OPENSTACK_URI | grep HTTP
```

Comment:
What are we doing here?   There's no mention of RHCOS_OPENSTACK_URI anywhere else in the doc.  Please clarify

Reply:
Removing the $RHCOS_OPENSTACK_URI

Reply:
Removing $RHCOS_QEMU_URI and mapping them with proper env variables.